R/mod_query_add.R

```diff
@@ -182,7 +182,7 @@ mod_query_add_server <- function(
       if(identical(new_query, query_in_db)){
         r$query_data <- dplyr::bind_rows(r$query_data, new_query)
       }
-      query_in_memory <- r$query_data[nrow(r$query_data), ]
+      query_in_memory <- r$query_data[nrow(r$query_data), -1]
```

Comment:
Yes, quick fix for the R CMD check. The tests for `all_review_data` was further modified to account for the new database table structure.

Reply:
It will be changed in the future when/if the query table gets redesigned. I thought adding the primary key for now would be helpful, but that's all that has changed.